### PR TITLE
Discount refactor button

### PR DESCRIPTION
**Pull request by meliajm**
We need to still be able to merge, so we will fix that in the morning
thanks!

### CODE REVIEW

File: rss-client/src/app/Quiz/employee/invalid-quiz/dialog-content.component 2.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div style="display: flex; flex-direction: column; align-items: center; justify-content: center; text-align: center;">
+    <h2 style="font-size: 70px; font-weight: 400;">You will not earn points for this quiz.</h2>
+    <!-- <mat-dialog-actions>
+        <button mat-raised-button color="warn" mat-dialog-close>I UNDERSTAND</button>
+    </mat-dialog-actions> -->
+</div>
```
Comment (GattieTime) [inline rss-client/src/app/Quiz/employee/invalid-quiz/dialog-content.component 2.html:6]: why comment? Didn't we remove material?

File: rss-client/src/app/User/Admin/admin-page/admin-page.component.html [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+        <ng-template [ngSwitchCase]="'quiz'">
+            <admin-quiz (navigateEvent)="navigateTo($event)"></admin-quiz>
+        </ng-template>
+        <ng-template [ngSwitchCase]="'inventory'">
+            <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
+        </ng-template>
+        <!--default case when there are no matches -->
+        <ng-template [ngSwitchDefault]>
+            <!-- <div class="container">
       <!- quizzes
```
Comment (GattieTime) [inline rss-client/src/app/User/Admin/admin-page/admin-page.component.html:15]: why spacing changes?

**Review by GattieTime — CHANGES_REQUESTED** [review]
There seem to be a lot of duplicate files and currently there are merge conflicts with dev, please resolve.

File: rss-client/src/app/User/Admin/admin-page/admin-page.component.html [hunk 3]
```diff
@@ -8,10 +8,10 @@
...
         <ng-template [ngSwitchCase]="'inventory'">
             <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
-        </ng-template>
-        <!--default case when there are no matches -->
-        <ng-template [ngSwitchDefault]>
-            <!-- <div class="container">
+    </ng-template>
+    <!--default case when there are no matches -->
+    <ng-template [ngSwitchDefault]>
+    <!-- <div class="container">
```
Comment (GattieTime) [inline rss-client/src/app/User/Admin/admin-page/admin-page.component.html:14]: Still not sure why the indent changes; also where are these divs closed?

**Review by GattieTime — COMMENTED** [review]
Please address question in specific comment.

File: rss-client/src/app/User/Admin/admin-page/admin-page.component.html (hunk 3 above)
Comment (meliajm) [reply rss-client/src/app/User/Admin/admin-page/admin-page.component.html:14]: there is not a good reason for the indent changes, i don't know why did those got moved
there are closing divs on lines 25 and 26 I believe

**Review by GattieTime — APPROVED** [review]

**Pull request merged**